libs/core/src/integration/email.schemas.ts

```diff
@@ -20,17 +20,17 @@ export const EnrichedUserMentionedNotification =
 
 export const EnrichedCommunityStakeNotification =
   CommunityStakeNotification.extend({
-    community_icon_url: z.string(),
+    community_icon_url: z.string().nullish(),
```

Comment:
Just making a note, this data is used in emails so if an icon_url is not set for a community we should use the same method as in the client to generate a default image:
```
        <div className={getClasses<{ size: IconSize }>({ size }, 'no-image')}>
          <CWText
            type={sizeIsAboveLarge ? 'h5' : 'caption'}
            className="avatar-no-image-letter"
            fontWeight="medium"
          >
            {community?.name?.slice(0, 1)}
          </CWText>
        </div>
```
in `cw_commuity_avatar.tsx`

Reply:
I don't necessarily disagree but the problem is will also have a lot of places to clean up due to strictNullChecks and our existing code not being designed for null checks.

I don't now where to place all the defaults so we're just going to have to raise them as bugs when they come up.

@timolegros Are you saying you want me to fix this or just that I should create a ticket for it? Do we have a default image?

Reply:
I'm not entirely sure, we can either generate a default image or do this HTML/CSS stuff in the email template itself. I'm not really sure how generating the image would work though. Can you create another ticket for it for now and we will worry about it later.

Reply:
OK.  Created!

https://github.com/hicommonwealth/commonwealth/issues/8519